python/py-numpy/Portfile

```diff
@@ -122,8 +122,11 @@ if {${name} ne ${subport}} {
 
         # set absolute path to remove references to @rpath/libmkl_rt.2.dylib
         post-destroot {
-            system "install_name_tool -change @rpath/libmkl_rt.2.dylib ${python.prefix}/lib/libmkl_rt.2.dylib ${destroot}${python.pkgd}/numpy/core/_multiarray_umath.cpython-${python.version}-darwin.so"
-            system "install_name_tool -change @rpath/libmkl_rt.2.dylib ${python.prefix}/lib/libmkl_rt.2.dylib ${destroot}${python.pkgd}/numpy/linalg/lapack_lite.cpython-${python.version}-darwin.so"
+            foreach dirname [glob -types d ${destroot}${python.pkgd}/numpy/*] {
```

Comment:
Hmm ... would love to use `fs-traverse` but it's about as complicated as this double `foreach`. A single `find` here would end up with a single command, which is nice: `[exec find ${destroot}${python.pkgd}/numpy "*.so"]`. I use this extensively in `qt4-mac`, for better or worse ... but it works very well & would do so here too.

Reply:
to me the double `foreach` isn't that complicated and clearly shows what's happening: go through all the subdirectories and then find the `.so` files and do something with them. The `exec find` would work I am sure, but it saves only one line of code... This code fixed the issue for me so I am all for now; since you're the maintainer feel free to use whichever way you prefer. You can either push to this PR or just commit it directly and close this - up to you, or let me know.

Reply:
Makes sense. I like the `exec find` better, so I'm testing that & will push direct & then note this PR and close this out. Thx!